src/tests/App.test.js

```diff
@@ -1,8 +1,9 @@
-import { render, screen } from '@testing-library/react';
-import App from '../components/App';
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import App from "../components/App";
 
-test('renders learn react link', () => {
+test("renders title", () => {
```

Comment:
consider adding more tests, for example testing the navbar to see if it has all the text. You could also try looking into fireEvents and see if you can work out how test that clicking the links on the navbar works.